appengine-skeleton-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/logging.properties

```diff
@@ -1,16 +1,13 @@
-#set( $symbol_pound = '#' )
-#set( $symbol_dollar = '$' )
-#set( $symbol_escape = '\' )
-${symbol_pound} A default java.util.logging configuration.
-${symbol_pound} (All App Engine logging is through java.util.logging by default).
-${symbol_pound}
-${symbol_pound} To use this configuration, copy it into your application's WEB-INF
-${symbol_pound} folder and add the following to your appengine-web.xml:
-${symbol_pound} 
-${symbol_pound} <system-properties>
-${symbol_pound}   <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
-${symbol_pound} </system-properties>
-${symbol_pound}
+# A default java.util.logging configuration.
+# (All App Engine logging is through java.util.logging by default).
+#
+# To use this configuration, copy it into your application's WEB-INF
+# folder and add the following to your appengine-web.xml:
+# 
+# <system-properties>
+#   <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
+# </system-properties>
+#
 
-${symbol_pound} Set the default logging level for all loggers to WARNING
```

Comment:
If you stop using them, you should not filter these files. See archetype-metadata.xml for details.